packages/app/package.json

```diff
@@ -13,7 +13,7 @@
         "redux-mock-store": "^1.5.4"
     },
     "dependencies": {
-        "@dhis2/analytics": "^21.8.2",
+        "@dhis2/analytics": "21.9.0",
```

Comment:
@kabaros should this be pinned to `21.9.0` or should it be an open range `^21.9.0`?  I think the latter for consistency unless there's a strong reason to pin the version